src/app/drive/components/FileViewer/FileViewerWrapper.tsx

```diff
@@ -157,18 +162,23 @@ const FileViewerWrapper = ({
     }
     return [];
   }, [folderFiles]);
-  const fileIndex = sortFolderFiles?.findIndex((item) => item.id === currentFile?.id);
-  const totalFolderIndex = sortFolderFiles?.length;
+
+  const folderItemsFiltered = folderItems
+    ? folderItems.filter((item) => !item.isFolder || item.type !== 'folder')
+    : undefined;
```

Comment:
nit: could simplify this to `folderItems?.filter((item) => !item.isFolder || item.type !== 'folder')`